pycheribuild/projects/cross/llvm.py

```diff
@@ -589,7 +589,7 @@ class BuildCheriOSLLVM(BuildLLVMMonoRepoBase):
     hide_options_from_help = True
 
     def configure(self, **kwargs):
-        self.add_cmake_options(LLVM_TARGETS_TO_BUILD="Mips;host")
+        self.add_cmake_options(LLVM_TARGETS_TO_BUILD="RISCV;Mips;host")
```

Comment:
Sort

Reply:
Alphabetically?

Reply:
Yes, though host is always at the end in the file (and I note that for Morello LLVM we put ARM before AArch64... booo)